interface_config.js

```diff
@@ -153,9 +153,11 @@ var interfaceConfig = {
     SHOW_BRAND_WATERMARK: false,
 
     /**
-    * Decides whether the chrome extension banner should be rendered on the landing page and during the meeting.
-    * If this is set to false, the banner will not be rendered at all. If set to true, the check for extension(s)
-    * being already installed is done before rendering.
+    * Decides whether the banner for the Jitsi Meetings Chrome extension
+    * (https://chrome.google.com/webstore/detail/jitsi-meetings/kglhbbefdnlheedjiejgomgmfplipfeb)
```

Comment:
This is not correct. That particular extension is only configured to meet.jit.si deployment and the value `kglhbbefdnlheedjiejgomgmfplipfeb ` is not hardcoded anywhere in the code.

Reply:
Great, that proves [my point](https://github.com/jitsi/jitsi-meet/issues/7163). Please clarify correctly. I tried to help but I'm new to Jitsi.

Reply:
The config for the banner is here: https://github.com/jitsi/jitsi-meet/blob/b4ecef429ada7181c1ac3f530c49ba43da2f6e1c/config.js#L419

Thanks for your continuous improvements to this @dandv !